events/emitter.go

```diff
@@ -38,13 +39,19 @@ func (e *Emitter) Post(ctx context.Context, evt Event) error {
 	return nil
 }
 
-func (e *Emitter) Events(ctx context.Context, clientID string) chan *event.Envelope {
+func (e *Emitter) Events(ctx context.Context, clientID string, fs ...filters.Filter) chan *event.Envelope {
+	eventFilters := filters.Filter(filters.Any(fs))
```

Comment:
Don't mind this comment, I keep forgetting that each filter is technically an array too